test/unit/models/flag.test.js

```diff
@@ -0,0 +1,46 @@
+const chai = require("chai");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+
+const addUser = require("../../utils/addUser");
+const cleanDb = require("../../utils/cleanDb");
+
+const flagModel = require("../../../models/flag");
+const firestore = require("../../../utils/firestore");
+const flagFirestore = firestore.collection("featureFlags");
+
+const flagData = {
+  title: "dark - mode",
+  enabled: true,
+  roles: ["members", "no_role", "super_user"],
+  users: {
+    "pallab id": true,
+    "rohit id": false,
+  },
+};
+
+chai.use(chaiHttp);
+
+describe("flags", function () {
+  beforeEach(async function () {
+    await addUser();
+  });
```

Comment:
Can we move `addFlag` inside `beforeEach`? This will make it easier for other people in future to write more feature flag tests.

```suggestion
describe("flags", function () {
  let flagId;
  beforeEach(async function () {
    flagId = await flagModel.addFlag(flagData);
  });
```